tests_python/test_release_modes.py

```
@@ -0,0 +1,154 @@
+    event_name: str
+    payload: dict[str, object]
+    expected: dict[str, str]
+
+
+@pytest.fixture(scope="module")
+def release_modes_module() -> ModuleType:
+    """Load the release mode helper as a Python module for unit tests."""
+
+    spec = util.spec_from_file_location("determine_release_modes", SCRIPT_PATH)
+    if spec is None or spec.loader is None:
+        raise RuntimeError("Unable to load determine_release_modes module")
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)